### PR TITLE
kmscon service: add extraOptions option

### PR DESCRIPTION
**Pull request by edef1c**
I needed this to add a couple options — perhaps this'll be useful to others too.

### CODE REVIEW

**Comment by shlevy** [comment]
Hmm, when I see `extraOptions` I think this will be added to the config file. Maybe `extraFlags` or similar?

File: nixos/modules/services/ttys/kmscon.nix [hunk 1]
```diff
@@ -32,6 +32,13 @@ in {
         default = "";
         example = "font-size=14";
       };
+
+      extraOptions = mkOption {
+        description = "Extra flags to pass to kmscon.";
+        type = types.string;
```
Comment (shlevy) [inline nixos/modules/services/ttys/kmscon.nix]: This should probably be a `types.listOf types.str`, and then we space-separate the options in `ExecStart`

Comment (nathan7) [inline nixos/modules/services/ttys/kmscon.nix]: So far I've seen `types.string`, `types.separatedString " "` and `types.listOf types.str`.
I can definitely change it, though.

**Comment by nathan7** [comment]
@shlevy How about this?

**Comment by nathan7** [comment]
In regard to `extraOptions` vs other names, most services seem to use `extraOptions` as the option name for this.

**Comment by nathan7** [comment]
The Travis failure appears to be due to a syntax error in the Travis script (?!), not due to an actual issue with my PR.

**Pull request merged**